lineman/app/components/thread_page/proposal_positions_panel/proposal_positions_panel.coffee

```diff
@@ -5,6 +5,8 @@ angular.module('loomioApp').directive 'proposalPositionsPanel', ->
   replace: true
   controller: ($scope, Records, CurrentUser, ModalService, VoteForm) ->
 
+    $scope.undecidedPanelOpen = false
```

Comment:
Oh, I was just looking at the view for this (`proposal_positions_panel/proposal_positions_panel.haml`) and don't see any reference to showUndecided / undecidedPanelOpen, so I thought these might be vestigial.